### PR TITLE
add javascript course to Romanian books

### PR DESCRIPTION
**Pull request by Aquaryus01**
## What does this PR do?
Add resource

## For resources
Javascript course

### Description
Add some Romanian javascript course in the Romanian books sections

### Why is this valuable (or not)?
It contains the basic stuff in Javascript

### How do we know it's really free?
You can access the website and no pay is needed.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course

## Checklist:
- [ X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ X] Include author(s) and platform where appropriate.
- [ X] Put lists in alphabetical order, correct spacing.
- [ X] Add needed indications (PDF, access notes, under construction).
- [ X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-ro.md [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
+
+
+### HTML and CSS
+
+* [HTML](http://tutorialehtml.com/ro/introducere-in-html/)
+
+
+### MySQL
+
+* [MySQL](http://profs.info.uaic.ro/~busaco/teach/courses/net/docs/mysql-ro.pdf) (PDF)
```
Comment (shrn01) [inline books/free-programming-books-ro.md:28]: why are we getting all these changes when the content is the same?

File: books/free-programming-books-ro.md [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
-
-
-### Scratch
-
-* [Informatica Creativa](http://scratched.gse.harvard.edu/resources/informatica-creativa-0)
+### Index
+
+* [Ajax](#ajax)
+* [C](#c)
+* [HTML and CSS](#html-and-css)
```
Comment (shrn01) [inline books/free-programming-books-ro.md:5]: Add javascript entry

**Review by shrn01 — CHANGES_REQUESTED** [review]

File: books/free-programming-books-ro.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-ro.md:28]: yes, very odd

File: books/free-programming-books-ro.md (hunk 2 above)
Comment (eshellman) [reply books/free-programming-books-ro.md:5]:
```suggestion
* [HTML and CSS](#html-and-css)
* [Javascript](#javascript)
```

File: books/free-programming-books-ro.md (hunk 1 above)
Comment (eshellman) [reply books/free-programming-books-ro.md:29]: perhaps encoding?

**Pull request merged**